syntax = "proto3";

import "svg.proto";

package transport_catalogue_proto;
        
message RenderSettings {
	double width = 1;
	double height = 2;
	double padding = 3;
	double stop_radius = 4;
	double line_width = 5;
	double underlayer_width = 6;
	uint32 bus_label_font_size = 7;
	uint32 stop_label_font_size = 8;
	repeated double bus_label_offset = 9;
	repeated double stop_label_offset = 10;
	Color underlayer_color = 11;
	repeated Color color_palette = 12;
}
